src/components/Autocomplete/components/ComboBox/tests/ComboBox.test.tsx

```diff
@@ -569,3 +585,8 @@ function dispatchKeyup(key: Key) {
   const event: KeyboardEventInit & {keyCode: Key} = {keyCode: key};
   document.dispatchEvent(new KeyboardEvent('keyup', event));
 }
+
+function dispatchKeydown(key: Key) {
```

Comment:
If we changed the above function signature to 
```
function dispatchKeyBoardEvent(key: Key, keyBoardEvent: string = 'keyup') {
```
we could reduce the code slightly.

Reply:
The issue here is that EventListener works off `window` events whereas KeyboardListener works off `document` events. So not sure its worth trying to make this more generic

Reply:
Gotcha didn't notice that. Ignore this comment 🙈